### DISCUSSION TITLE
Tileset3D props passed from Tile3DLayer

### DISCUSSION THREAD

**Discussion by radoslavirha** [visgl/deck.gl#7729] [Q&A]
Hi,

I was trying to play with Tile3DLayer performance by passing some optimization params as mentioned in [docs](https://deck.gl/docs/api-reference/geo-layers/tile-3d-layer#loadoptions). But I can't see any difference. So I dug in code.

Here
https://github.com/visgl/deck.gl/blob/4530b327ed712f75f95930a94073409d1b516246/modules/geo-layers/src/tile-3d-layer/tile-3d-layer.ts#L200

When I log options passed to Tileset3D, I can see `loadOptions` passed to Tile3DLayer:
```
{
  "cesium-ion": { accessToken: ION_TOKEN },
  tileset: {
    throttleRequests: false
  }
}
```

Which seems to be ok with Tileset3DProps type
https://github.com/visgl/loaders.gl/blob/39640ebd195c6e02bd8d50e60ae39f506dc08df1/modules/tiles/src/tileset/tileset-3d.ts#L65

For me it's little bit confusing it accepts optimization params + `loadOptions`

But here
https://github.com/visgl/loaders.gl/blob/39640ebd195c6e02bd8d50e60ae39f506dc08df1/modules/tiles/src/tileset/tileset-3d.ts#L273

It merges my `loadOptions` with defaults

When speaking about `throttleRequests` param from `loadOptions.tileset` it seems to be unused

https://github.com/visgl/loaders.gl/blob/39640ebd195c6e02bd8d50e60ae39f506dc08df1/modules/tiles/src/tileset/tileset-3d.ts#L305

^^ is always used default value and not destructured value from `loadOptions.tileset`

Am I right?

Here is [example code](https://codesandbox.io/s/deck-gl-3d-tiles-drilg0?file=/src/App.js) where I used

```
tileset: {
  show: false
}
```

which should not show data?

Basically I think `loadOptions.tileset` is not correctly passed to Tileset3D, or not correctly destructured/used in Tileset3D

Thanks

**Comment by bortomar** [comment]
Hello, same issue here. Any progess?

**Comment by radoslavirha** [comment]
Hello, had anybody chance to check it out? Thanks
